src/test/scala/mesosphere/marathon/SchedulerActionsTest.scala

```diff
@@ -40,19 +41,19 @@ class SchedulerActionsTest extends TestKit(ActorSystem("TestSystem")) with Marat
     when(repo.expunge(app.id)).thenReturn(Future.successful(Seq(true)))
     when(taskTracker.get(app.id)).thenReturn(Set.empty[Protos.MarathonTask])
 
-    queue.rateLimiter.addDelay(app)
-
-    queue.rateLimiter.getDelay(app).hasTimeLeft should be(true)
+    //    queue.rateLimiter.addDelay(app)
```

Comment:
Code is commented out, but the test name suggests it should be part of the test?